### PR TITLE
Added ui tests for navigating to settings & help page

### LINKED ISSUE #1032: Add main navigation ui tests for menu items
Continuing with this [PR](https://github.com/woocommerce/woocommerce-android/pull/316) and adding some additional ui navigation tests for menu items:

- [ ] verify that when settings clicked, menu is displayed
- [ ] verify there are only 2 items in menu
- [ ] verify that when first item is clicked, it opens settings
- [ ] verify that when 2nd item is clicked, it opens help screen

### PR DESCRIPTION
**Pull request by anitaa1990**
Fixes #1032 

Adds the following UI navigation tests for MainActivity:
- [x] verify that when settings clicked, menu is displayed
- [x] verify there are only 2 items in menu
- [x] verify that when first item is clicked, it opens settings
- [x] verify that when 2nd item is clicked, it opens help screen

### To Test:
- [x] Run MainNavigationTest

### CODE REVIEW

File: WooCommerce/src/androidTest/kotlin/com/woocommerce/android/helpers/WCMatchers.kt [hunk 1]
```diff
@@ -23,4 +24,17 @@ object WCMatchers {
             }
         }
     }
+
+    fun correctNumberOfItems(itemsCount: Int): Matcher<View> {
```
Comment (nbradbury) [inline WooCommerce/src/androidTest/kotlin/com/woocommerce/android/helpers/WCMatchers.kt]: This function could use a comment describing its purpose.

**Comment by nbradbury** [comment]
@anitaa1990 The added tests are passing, but looks like `dashboardMenuOptionDisplaysDashboardView` is failing because it assumes the dashboard toolbar title is `R.string.my_store`. We changed this to the store name a little while back.

We can address this in a separate PR, but thought I'd mention it just in case you wanted to tackle it here.

**Comment by anitaa1990** [comment]
@nbradbury that's what I thought so at first too. But then noticed that in [here](https://github.com/woocommerce/woocommerce-android/blob/develop/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/dashboard/DashboardFragment.kt#L213
), we are returning the default store name if the site does not exist. In this case, the site is mocked so it actually returns the 'My store' string by default.

So the test was not failing because of that but because of this issue #1030 . This is how I noticed that something was up :)

**Comment by nbradbury** [comment]
> So the test was not failing because of that but because of this issue #1030 . This is how I noticed that something was up :)

Ah, nice catch! I pulled #1030 into this locally and can confirm it fixes the test, so we can ignore the failure here.

File: WooCommerce/src/androidTest/kotlin/com/woocommerce/android/helpers/WCMatchers.kt (hunk 1 above)
Comment (anitaa1990) [reply WooCommerce/src/androidTest/kotlin/com/woocommerce/android/helpers/WCMatchers.kt]: Oh, nice catch! I just noticed that the method name provides no context as well. Updated both 👍

Comment (nbradbury) [reply WooCommerce/src/androidTest/kotlin/com/woocommerce/android/helpers/WCMatchers.kt]: Not sure what happened, but I'm not seeing any changes to `WCMatchers` in your commit to fix this (80e96e0).

Comment (anitaa1990) [reply WooCommerce/src/androidTest/kotlin/com/woocommerce/android/helpers/WCMatchers.kt]: Oh man, I seem to have checked in only one file 😟. Sorry about the confusion!

**Comment by nbradbury** [comment]
:shipit:

**Pull request merged**